source/features/mark-private-orgs.tsx

```diff
@@ -22,9 +22,10 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	const publicOrganizations = await getPublicOrganizations(getUsername());
+	const publicOrganizationsNames = await getPublicOrganizationsNames(getUsername());
 	for (const org of orgs) {
-		if (!publicOrganizations.includes(org.pathname.slice(1))) {
+		// https://github.com/sindresorhus/refined-github/issues/2830#issuecomment-591111897
+		if (!publicOrganizationsNames.includes(org.pathname.slice('/organizations/'.length))) {
```

Comment:
Looks like a typo here, you're missing a ) after organizations.

Reply:
It's to slice the value of `'/organizations/'.length`, equivalent to `.slice(15)`. Did that to enhance readability (not to have to guess where the 15 comes from, but now it leads to a syntax error assumption. The irony 😄

Reply:
I see, I was thinking it was an array at a quick glance, but it's a string. This approach won't be backwards compatible with GHE, I'm assuming it's rolled out for all of GH, and ideally we remove `/organizations/` in another way that would work even if it exists or not.

Reply:
Addressed via ff49bdb.